project_liberation/management/commands/load_initial_data.py

```diff
@@ -113,13 +124,15 @@ def handle(self, *args, **options):
                 body=body,
                 author=author,
                 read_time=random.randint(1, 10),
+                recommended_articles=recommended_articles,
                 views=0,
                 cover_photo=wagtail_cover_photo,
                 article_photo=wagtail_article_photo,
                 is_main_article=True,
             )
             blog_index_page.add_child(instance=blog_article_page)
             blog_article_page.save()
+            last_id = blog_article_page.id
 
```

Comment:
When I'm looking right now on this file I see how bad it is written and should be divided for small methods. My bad :P